transformers/asteroids

```diff
@@ -0,0 +1,23 @@
+const { getAverageValue } = require('../helpers/math');
+
+const transformAsteroidsData = (asteroidsData) => ({
```

Comment:
this is not critical, but such transformations are usually called mappers

Reply:
Thanks. Remaned.